docs/monitoring/reports-and-components.md

```diff
@@ -47,6 +47,13 @@ Each alert query is visible in the Alert Rules list.
 
 By checking out the [Integrations documentation](https://sematext.com/docs/integration/) you can learn more about how to monitor various programming languages, operating systems, containers (Docker, Kubernetes, Mesos), Cloud IaaS / PaaS services, iOS, Android, AWS EC2, AWS S3, Log Management, and log shippers. There you will find detailed information on how to install and configure different agents required to start sending your logs and metrics data to Sematext Cloud.
 
+### Groups
+Report components can be grouped by tags using the `Groups` drop down menu in the Monitoring Report Menu. One of the [available tags](https://sematext.com/docs/tags/common-schema/) for grouping is the `os.kernel` tag, which will put hosts running the same Linux kernel in the same group. This can be useful to check whether some hosts need to be upgraded with a newer kernel.
```

Comment:
Report components ==> Data (dataseries?) in report components
drop down menu ==> drop-down? 
Monitoring Report Menu ==> monitoring report menu (because this is not a name, or is it?)

Re os.kernel.... not sure why this is brought up here?  Is it used simply as an example?

Note that this makes sense, but it's referring to data in *this particular* table.  Most tables ignore grouping, right?

But grouping is also often used for grouping data in timeseries charts.  Maybe we want to over that with text + screenshot?

Reply:
> Monitoring Report Menu ==> monitoring report menu (because this is not a name, or is it?)

We use it in this page as a name.

> Re os.kernel.... not sure why this is brought up here? Is it used simply as an example?

Yes, this is an example

> Note that this makes sense, but it's referring to data in this particular table. Most tables ignore grouping, right?

No, actually most Infra components (overview tables & charts) have grouping.

> But grouping is also often used for grouping data in timeseries charts. Maybe we want to over that with text + screenshot?

Added an example for grouping using `os.disk` tag in storage charts.